src/redux/migrations.ts

```diff
@@ -755,4 +755,11 @@ export const migrations = {
       selectFiatConnectQuoteLoading: false,
     },
   }),
+  68: (state: any) => ({
```

Comment:
I think the migration-related code will need an update once you merge main into this PR